site/package.json

```diff
@@ -7,13 +7,13 @@
   "scripts": {
     "build": "yarn exe scripts/build.ts",
     "clean": "rimraf ./.next/",
+    "codegen": "yarn exe scripts/codegen.ts",
```

Comment:
Extracted from `prepare`.  `patch-package` is called from the root `package.json` prepare for simplicity. This step is temporary anyway and will be replaced with [native patches](https://yarnpkg.com/cli/patch) in yarn 3.